src/index.js

```diff
@@ -0,0 +1,66 @@
+import './style.css';
+import localstore from './localstorage.js';
+import completed from './completedtasks.js';
+import displayhtml from './htmldisplay.js';
+
+const todoList = [];
+
+const renderUI = () => {
+  if (!JSON.parse(localStorage.getItem('todo-list-group'))) {
+    localstore.storetask(localstore.sortindex(todoList));
+  }
+  // ***Displaying the List of tasks in UI*****
+  displayhtml.displayList();
+  // ***Event for Checkboxes***
+  const checkboxes = document.querySelectorAll('.checkbox');
+
+  [...checkboxes].forEach((button) => {
+    button.addEventListener('change', completed);
+  });
+  // ***Event for Update Task****
+  const inputs = document.querySelectorAll('.description');
+  [...inputs].forEach((input) => {
+    input.addEventListener('focusout', localstore.updateTask);
+  });
```

Comment:
- [ ] consider using the arrow function or the traditional function, that will make the code much simpler and will help you significantly in the code testing part.